echo/index.js

```diff
@@ -115,6 +119,17 @@ function Client() {
     return;
   }
 
+
+  var getErrorCode = function (err, statusCode) {
+    if (err) {
+      return errorCodesAndLabels.REQUEST_ERROR;
+    } else if (httpStatusToErrorCode[statusCode]) {
+      return httpStatusToErrorCode[statusCode];
+    } else {
```

Comment:
This `else` isn't needed. Just to the final `return` after the final `else if` block

Reply:
sure!